### ISSUE TITLE
Feil: aria-required-parent feiler i Select.

### ISSUE THREAD

**Issue by lfbergee** [fremtind/jokul#1135]
**Feilbeskrivelse**
Ved oppdatering av dependecies finner axe-core en feil med aria-håntering av Select.

```
  Expected the HTML found at $('#jkl-select-17QkQxvH__2') to have no violations:

    <button type="button" id="jkl-select-17QkQxvH__2" class="jkl-select__option" data-testid="jkl-select__option" aria-selected="false" role="option" value="2">2</button>

    Received:

    "Certain ARIA roles must be contained by particular parents (aria-required-parent)"

    Fix any of the following:
      Required ARIA parent role not present: listbox

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/3.5/aria-required-parent?application=axeAPI
```

Reglen er skrudd av for å få inn oppdateringen, bør skrus på igjen når dette issue hånteres.

**For å reprodusere**

1. Gå til select.test.ts
2. Skru på regelen igjen 'aria-required-parent'
3. Kjør tester

**Comment by piofinn** [comment]
Det kan nok hende det riktigste her er å fjerne `role="option"` fra valgene. `Core toggle` setter `role="group"` på hovedelementet ved render, og i følge deres tilgjengelighetsekspert gir det en bedre opplevelse for skjermlesere. Se mer i PRen jeg åpnet om dette hos NRK for lenge siden: https://github.com/nrkno/core-components/pull/414

**Comment by wkillerud** [comment]
Prøvde fjerne role="option", men det ødela tastaturnavigasjon. Begynner å bli en stund siden diskusjonen med NRK, men så ikke ut som det ble foreslått noen alternativ løsning. For hva det er verdt: `axe` klager ikke lenger på dette issuet om jeg fjerner unntakene. Kanskje bare pushe en endring som fjerner unntakene og si oss fornøyde @piofinn?
